core/spec/models/spree/calculator/shipping/price_sack_spec.rb

```diff
@@ -14,22 +14,31 @@
 
   it_behaves_like 'a calculator with a description'
 
-  let(:order) { stub_model(Spree::Order) }
-  let(:shipment) { stub_model(Spree::Shipment, amount: 10) }
+  describe '#compute' do
+    subject { calculator.compute(package) }
+    let(:package) { build(:stock_package, variants_contents: { build(:variant) => 1 }) }
 
-  # Regression test for https://github.com/spree/spree/issues/714 and https://github.com/spree/spree/issues/739
-  it "computes with an order object" do
-    calculator.compute(order)
-  end
+    before do
+      # This hack is due to our factories not being so smart to understand
+      # that they should create line items with the price of the associated
+      # variant by default.
+      allow_any_instance_of(Spree::Stock::ContentItem).to receive(:price) { amount }
```

Comment:
Do you think it would make sense (be possible, simple,  and useful for other scenarios) to change the factory in order to avoid this stubbing?

Reply:
I had the same thought but I think we have to change the line item price having a dynamic price, based on the variant price. I think that this would break a lot of specs in stores that are relying on the static price of line items in our core factories, and that's why I think this is the best trade-off. 

I'm open to making an attempt to create an alternative line_item factory with a dynamic price and trying to deprecate the current one. Do you think it's worth it?

Reply:
I think it is the normal scenario for line items to have prices bound to the ones of their variants... I would consider a price difference to be more of an edge case, so I think that factory could be later changed to reflect a more regular use case. 
But of course I would not consider this an urgent matter at all. We may just open an issue and see how it goes from there.

Reply:
Agree, let me do that!